packages/connect-examples/new-example/app/components/documentation/ArchitectureDiagram.tsx

```
@@ -0,0 +1,299 @@
+  return (
+    <>
+      <div className="relative w-full h-[450px] bg-muted/30 border border-border/50 rounded-lg not-prose overflow-hidden">
+        <ReactFlow {...flowProps} />
+        <Button
+          variant="ghost"
+          size="icon"
+          className="absolute top-2 right-2 h-8 w-8"
+          onClick={() => {
+            window.open(
+              window.location.href + '?diagram=fullscreen',
+              '_blank',
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)